src/DataDictionary/utils.js

```diff
@@ -107,6 +113,11 @@ export const downloadMultiTemplate = (format, nodesToDownload) => {
       throw new Error(`cannot download template for node "${nodeID}"`);
     });
   })).then(() => {
+    const time = new Date();
+    const startingNodeName = 'project';
+    let readmeContent = `The following TSV templates were downloaded from ${appname} on ${time.toLocaleDateString()} ${time.toLocaleTimeString()}. The following are all possible paths from ${startingNodeName} node to ${clickingNodeName} using data dictionary version ${dictionaryVersion}. The downloaded ${format} files need to be submitted in the order shown in the chosen path(s) below.\n`;
```

Comment:
`The following TSV templates were downloaded` should be `These ${format} templates were downloaded`

Reply:
Also, I think we could either add a number for each path in the list below, or add something like `There are {$totalPathNumber} paths in total.` before ` The downloaded ${format} files need to be submitted in the order shown in the chosen path(s) below.` in here.

Reply:
thanks! will add path number~